.github/workflows/docker.yml

```diff
@@ -3,6 +3,9 @@ name: Build Docker Images
 on:
   push:
     branches: master
+  tags:
+    - "[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
Is this also triggered for alpha releases?

Reply:
Yes, if we only wanted stable releases, the regex would be without the asterix.